storage/blackhole/ha_blackhole.cc

```diff
@@ -363,6 +394,16 @@ static int blackhole_fini(void *) {
 struct st_mysql_storage_engine blackhole_storage_engine = {
     MYSQL_HANDLERTON_INTERFACE_VERSION};
 
+static MYSQL_SYSVAR_BOOL(transactions, blackhole_transactions,
+                         PLUGIN_VAR_OPCMDARG,
+                         "Enables transaction support for Blackhole engine",
+                         /*check=*/nullptr,
+                         /*update=*/nullptr,
+                         /*default=*/blackhole_transactions);
```

Comment:
While this should just take value from the variable (`false`, but even that requires knowing that statics are zero-initialized), the sysvar definition looks circular superficially.
```suggestion
                         /*default=*/false);
```

Reply:
Makes sense. I initially tried to avoid having a default set in multiple places.